### ISSUE TITLE
[Rust] Implement new Concept Exercise: tuple-structs

### ISSUE THREAD

**Issue by efx** [exercism/rust#1067]
This issue describes how to implement the `tuple-structs` concept exercise for the Rust track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

Teach the student how to work with tuple structs in Rust. Tuple structs are also called named structs.

## Learning objectives

- know what tuple structs are
- know how to declare and access values in tuple structs

## Out of scope

- broad use cases for unit structs
- customizing representation in memory of structs
- unit structs

## Concepts

- tuple structs
- destructuring

## Prerequisites

- tuples
- structs

### Hints

- https://doc.rust-lang.org/book/ch05-01-defining-structs.html#using-tuple-structs-without-named-fields-to-create-different-types 
- https://doc.rust-lang.org/stable/rust-by-example/custom_types/structs.html

### After

- https://doc.rust-lang.org/book/ch05-01-defining-structs.html#using-tuple-structs-without-named-fields-to-create-different-types 
- https://doc.rust-lang.org/stable/rust-by-example/custom_types/structs.html

## Representer

No changes required

## Analyzer

No changes required

## Implementing

See our [implementation guide](https://github.com/exercism/v3/blob/master/languages/rust/howto/implementing-a-concept-exercise.md#how-to-implement-a-rust-concept-exercise) for guidance.  In addition you may want to make use of the shell script while in the `concept` directory, with the arg loops, e.g. `sh boil_exercise.sh tuple-structs`

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.
